server/backup_runner.go

```diff
@@ -34,8 +37,8 @@ func (s *BackupRunner) Run(ctx context.Context, wg *sync.WaitGroup) {
 	defer ticker.Stop()
 	defer wg.Done()
 	log.Debug("Auto backup runner started", zap.Duration("interval", s.backupRunnerInterval))
-	runningTasks := make(map[int]bool)
-	var mu sync.RWMutex
+	runningBackupTasks := make(map[int]bool)
+	var muBackup sync.Mutex
```

Comment:
Is this mutex used at the right place?

Reply:
It's not a new change. I'll revert the related renaming in this PR for clarity.